Sources/AltSwiftUI/Source/CoreUI/ViewBinder.swift

```diff
@@ -42,15 +42,41 @@ class ViewBinder {
         self.bodyLevel = bodyLevel
         self.isInsideButton = isInsideButton
         self.overwriteTransaction = overwriteTransaction
+
+        registerTextSizeChangedNotification()
     }
 
     func registerStateNotification(origin: Any) {
         NotificationCenter.default.removeObserver(self, name: Self.StateNotification.name, object: origin)
         NotificationCenter.default.addObserver(self, selector: #selector(handleStateNotification(notification:)), name: Self.StateNotification.name, object: origin)
     }
     
+    func registerTextSizeChangedNotification() {
+        NotificationCenter.default.removeObserver(self, name: UIContentSizeCategory.didChangeNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(handleTextSizeChangedNotification(notification:)), name: UIContentSizeCategory.didChangeNotification, object: nil)
+    }
+
     // MARK: - Private methods
-    
+
+    @objc private func handleStateNotification(notification: Notification) {
+        if notification.name == Self.StateNotification.name {
+            let transaction = notification.userInfo?[Self.StateNotification.transactionKey] as? Transaction
+            // TODO: Improves view update performance,
+            // but causes small delay. Need to find better way
+            // to queue without delay, before render happens.
+            // queueRenderUpdate(transaction: transaction)
+            updateView(transaction: transaction)
+        }
+    }
+    @objc private func handleTextSizeChangedNotification(notification: Notification) {
+        if notification.name == UIContentSizeCategory.didChangeNotification {
+            // TODO: Improves view update performance,
+            // but causes small delay. Need to find better way
+            // to queue without delay, before render happens.
+            // queueRenderUpdate(transaction: transaction)
+            updateView(transaction: nil)
```

Comment:
I think there is no guarantee the font will update from just updating the view, as the condition for fonts to update after a content size change is that the font object is recreated.

Please try with `adjustsFontForContentSizeCategory` first and explain any issue if there is.

Reply:
Thanks for your advice. The implementation has been updated.